drainer/config_test.go

```diff
@@ -18,12 +18,16 @@ import (
 	"io/ioutil"
 	"path"
 	"testing"
+	"time"
 
 	"github.com/BurntSushi/toml"
 	"github.com/coreos/etcd/integration"
 	. "github.com/pingcap/check"
 	"github.com/pingcap/parser/mysql"
+	dsync "github.com/pingcap/tidb-binlog/drainer/sync"
 	"github.com/pingcap/tidb-binlog/pkg/util"
+	"github.com/pingcap/tidb-binlog/pkg/zk"
+	gozk "github.com/samuel/go-zookeeper/zk"
```

Comment:
According to the [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#imports), we should rename the project-specific `zk`, not the more general one.